backend/migrations/000013_add_community_categories_and_toc.up.sql

```diff
@@ -22,4 +22,5 @@ INSERT INTO community_types (key, name, description)
 VALUES ('service', 'Service', 'Service Community');
 
 ALTER TABLE communities ADD COLUMN category community_type;
-ALTER TABLE communities ADD COLUMN terms_and_conditions_url VARCHAR(256);
\ No newline at end of file
+ALTER TABLE communities ADD COLUMN terms_and_conditions_url VARCHAR(256);
+ALTER TABLE communities ADD COLUMN only_authors_to_submit BOOLEAN;
```

Comment:
We shouldn't alter migrations that have already been applied to the database because the change here will never be applied to the production database. Instead these alter table commands should be performed in migration #15.

Reply:
thanks! Will fix.